pnpm-lock.yaml

```diff
@@ -52,7 +52,7 @@ dependencies:
   '@material-ui/core': 4.12.4_nylzxt5ale4dsv666zkb25cgtm
   '@material-ui/icons': 4.11.3_syzufrrnbgjhwvvotesawp6yai
   '@material-ui/lab': 4.0.0-alpha.61_syzufrrnbgjhwvvotesawp6yai
-  '@saleor/app-sdk': 0.16.0_c3hne4hwj64hb7tofigd3bvkji
+  '@saleor/app-sdk': link:../saleor-app-sdk/dist
```

Comment:
@lkostrowski seems you have changes in lock from local development

Reply:
yes, its dependent on sdk, changing to draft